### PR TITLE
New flags readme

### PR DESCRIPTION
**Pull request by nol166**
### Describe in detail the problem you had and how this PR fixes it
Update quick start guide and readme based on new flags introduced with #420 

Adds a clause in bug report template about ide.coder.com issues

### Is there an open issue you can link to?
n/a

### CODE REVIEW

File: doc/self-hosted/index.md [hunk 1]
```diff
@@ -37,24 +37,33 @@ code-server can be ran with a number of arguments to customize your working dire
...
+  -N, --no-auth               Start without requiring authentication.
+  -H, --allow-http            Allow http connections.
+  -P, --password <value>      Specify a password for authentication.
+  --bootstrap-fork <name>     Used for development. Never set.
+  --fork <name>               Used for development. Never set.
+  --extra-args <args>         Used for development. Never set.
+  -h, --help                  output usage information
   ```
+  ### Extension Directory
+  Point code-server at a local directory with installed extensions. This allows inheriting local VSCode extensions by specifying `--extensions-dir ~/.vscode/extensions`
```
Comment (kylecarbs) [inline doc/self-hosted/index.md]: Feels a bit difficult to parse this. Maybe:
```
Specify a custom directory for extensions. Provides compatibility with local VS Code extensions by specifying `--extensions-dir ~/.vscode/extensions`.
```

File: doc/self-hosted/index.md [hunk 2]
```diff
@@ -37,24 +37,33 @@ code-server can be ran with a number of arguments to customize your working dire
...
 
-  ### Data Directory
+  ### User Data Directory
+  Specify the root folder that VS Code will start in. This allows inheriting local VS Code configuration. Example `--user-data-dir ~/.config/User`
```
Comment (kylecarbs) [inline doc/self-hosted/index.md]: Maybe:
```
Specify where data is stored. Provides compatibility with local VS Code configuration by specifying `--user-data-dir ~/.config/User`.
```

File: README.md [hunk 3]
```diff
@@ -50,6 +50,9 @@ Quickstart guides for [Google Cloud](doc/admin/install/google_cloud.md), [AWS](d
 
 How to [secure your setup](/doc/security/ssl.md).
 
+### Customize User Data
+- Use your local VSCode configuration and locally installed extensions with the `--user-data-dir` and `--extensions-dir` flags. View the [quick start](doc/self-hosted/index.md) guide for more.
```
Comment (kylecarbs) [inline README.md]: Maybe:

### Use VS Code Data
Use your existing VS Code configuration and extensions using the `--user-data-dir` and `--extensions-dir` flags. View the [quick start](doc/self-hosted/index.md) guide for more.

**Comment by illiapoplawski** [comment]
The readme has the "-h" flag twice.  First for (-h, --host) and second for (-h, --help) causing a conflict as -h will never trigger for the help menu.  This conflict is also found in the latest release.

File: doc/self-hosted/index.md (hunk 2 above)
Comment (joelhock) [reply doc/self-hosted/index.md]: I realize this is correct for Linux, but for mac (at least on my installation), the local VS Code configuration is stored in `--user-data-dir ~/Library/Application\ Support/Code/`
It'd be nice to list both.

**Comment by illiapoplawski** [comment]
Just noticed doc/self-hosted/index.md also mentions the code-server -h help option at bottom.

Use `code-server -h` or` code-server --help` to view the usage for the cli. This is also shown at the beginning of this section.

should read:
Use `code-server --help` to view the usage for the cli. This is also shown at the beginning of this section.

**Comment by kylecarbs** [comment]
I've updated this in the telemetry PR

**Pull request closed without merging**